specification/gp-connect-access-record-fhir.yaml

```
@@ -213,4 +253,119 @@ servers:
     description: Integration test environment.
   - url: 'https://api.service.nhs.uk/gp-connect-access-record-fhir'
     description: Production environment.
-paths: {}
+paths: 
+  /FHIR/STU3/$access-record-structured:
```

this needs to be a FHIR operation on a Patient resource, as per https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured_development_retrieve_patient_record.html#fhir-relative-request

```suggestion
  /FHIR/STU3/Patient/$access-record-structured:
```